lib/quicly.c

```diff
@@ -209,6 +211,24 @@ struct st_quicly_conn_t {
          *
          */
         unsigned send_handshake_done : 1;
+        /**
+         *
+         */
+        int64_t send_ack_at;
+        /**
+         *
+         */
+        struct {
+            struct cc_var ccv;
+            size_t bytes_in_flight;
+            uint64_t end_of_recovery;
+            /**
+             * collected by on_ack_cc
+             */
+            struct {
+                size_t nsegs, nbytes;
```

Comment:
Do these vars refer to the number of packets/segs and bytes newly acknowledged?

Reply:
Yes.